sweep/fee_bumper.go

```diff
@@ -1122,6 +1122,16 @@ func (t *TxPublisher) handleInitialTxError(r *monitorRecord, err error) {
 
 		result.Event = TxFailed
 
+		// Calculate the starting fee rate to be used when retry
+		// sweeping these inputs.
```

Comment:
See my question before, does it mean "TxFailed" inputs are never grouped with new inputs ?

Reply:
nope they can be grouped with new inputs